lib/editor/components/map/LocationsLayer.js

```diff
@@ -164,8 +164,8 @@ export default class LocationsLayer extends PureComponent<Props, State> {
       layerType = isPolygon ? 'polygon' : 'polyline'
 
       const formattedEditedCoords = isPolygon
-        ? editing.latlngs[0][0].map(latlng => [latlng.lat, latlng.lng])
-        : editing.latlngs[0].map(latlng => [latlng.lat, latlng.lng])
+        ? editing.latlngs[0][0].map(convertLatLngToArray)
+        : editing.latlngs[0].map(convertLatLngToArray)
```

Comment:
See https://github.com/ibi-group/datatools-ui/pull/754/files/163742b390496bdd7ab7cff517bdf46360653a6d..dadf298f26d1c7bf9e1254c4ae18cedc5ae74a0a#r798778538. It is actually the exact same code.

Reply:
@philip-cline do you have any ideas on how best to refactor this

Reply:
Yep it's the same code, so a util function would make sense